CHANGELOG.md

```diff
@@ -1,7 +1,26 @@
-## next release
+## 2.3.0 / 2018-06-05
 
+* [FEATURE] Add query commands to promtool
+* [FEATURE] Add security headers to HTTP server responses
+* [FEATURE] Pass query hints via remote read API
 * [CHANGE] `marathon_sd`: use `auth_token` and `auth_token_file` for token-based authentication instead of `bearer_token` and `bearer_token_file` respectively.
+* [ENHANCEMENT] Optimise PromQL for memory usage and allocations
+* [ENHANCEMENT] Limit number of dropped targets in web UI
+* [ENHANCEMENT] Add advanced filterting configuration to Consul service discovery
```

Comment:
filtering

And it's passing filtering options down to Consul, this was all already possible via relabelling

Reply:
Yes, we still added advanced filtering to the consul configuration itself. Not sure how to phrase it differently that would change the underlying meaning significantly.